ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,104 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{ml_survey_port | default("7201")}} 
+
+#Application ENV example local/dev/production
+NODE_ENV={{ml_survey_node_env|default("production")}}
+
+#Log configuration
+#Enable console logging
+ENABLE_DEBUG_LOGGING={{ml_survey_enable_debug_logging | default("ON")}}
+
+#Enable writing logs to a file
+ENABLE_BUNYAN_LOGGING={{ml_survey_enable_bunyan_logging | default("OFF")}}
+
+#service based URL
+APPLICATION_BASE_URL={{ml_survey_application_base_url | default('/assessment/')}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{ml_survey_request_timeout_for_reports | default("600000")}}
+MONGODB_URL={{ml_survey_mongodb_url}}
+
+## Cloud Environment platform
+# which will be templatized from the above variable (defaults)
+
+# Azure Cloud Configuration
+CLOUD_STORAGE=AZURE
+AZURE_ACCOUNT_NAME={{ml_survey_azure_account_name}}
+AZURE_ACCOUNT_KEY={{ml_survey_azure_account_key}}
+AZURE_STORAGE_CONTAINER={{ml_survey_azure_storage_container)}}
+
+# Google Cloud Configuration
+# Optional
+#CLOUD_STORAGE=GCP
```

Comment:
why are we having this commented variables? if needed can you give some description?

Reply:
As per discussion with santhosh we are mandatory using the Azure, while in PR azure should be there, yes the description has been provide as GCP as optional, if any thing specific comment is required i will do that

Reply:
Please update the comment and mark this as resolved